.env

```diff
@@ -14,3 +14,7 @@ ORACLE_PASSWORD=guest
 # for db2
 DB2INST1_PASSWORD=root_pass
 LICENSE=accept
+
+# sybase user/pwd
+SYBASE_USER=sa
```

Comment:
I'm not sure for sybase, need to investigate more how to rewrite default credentials